tests/test_home_page.py

```diff
@@ -0,0 +1,7 @@
+from app import app
+
+
+def test_submit_data():
+    client = app.test_client()
```

Comment:
Is there a way to avoid making a client every time we want to test a functionality? Maybe we could have a test setup function if its possible.

Reply:
I believe this is possible using application factories, I'll take a look.

Reply:
Done.